Dockerfile

```
@@ -0,0 +1,56 @@
+# ================ #
+#    Base Stage    #
+# ================ #
+
+FROM node:18-alpine as base
+
+WORKDIR /usr/src/app
+
+ENV HUSKY=0
+ENV CI=true
+ENV LOG_LEVEL=info
+
+# RUN apk add --no-cache dumb-init
+
+COPY --chown=node:node yarn.lock .
+COPY --chown=node:node package.json .
+COPY --chown=node:node .yarnrc.yml .
+COPY --chown=node:node .yarn/ .yarn/
+
+RUN sed -i 's/"prepare": "husky install .github\/husky"/"prepare": ""/' ./package.json
+
+# ENTRYPOINT ["dumb-init", "--"]
+
+# ================ #
+#   Builder Stage  #
+# ================ #
+
+FROM base as builder
+
+ENV NODE_ENV="development"
+
+COPY --chown=node:node tsconfig.base.json .
+COPY --chown=node:node scripts/ scripts/
+COPY --chown=node:node src/ src/
+
+RUN yarn install --immutable
+RUN yarn run build
+
+# ================ #
+#   Runner Stage   #
+# ================ #
+
+FROM base AS runner
+
+ENV NODE_ENV="production"
+ENV NODE_OPTIONS="--enable-source-maps --experimental-fetch"
```

```suggestion
ENV NODE_OPTIONS="--enable-source-maps"
```

Unknown flag on Node 18